swagger.yaml

```diff
@@ -721,13 +727,78 @@ paths:
                 items:
                   $ref: "#/components/schemas/Error"
 
+  "/v1/users/{userGuid}/flextime": 
+    get:
+      tags:
+        - UsersRead
+      summary: 
+        Get the flextime balance for a user for a specified date. Total balance
+        is returned for the given date. Month balance is the balance for the
+        month of the given date. Values are returned only if the advanced time
+        tracking add-on is active.
+      operationId: Flextime_GetFlextime
```

Comment:
use camelCase and similar naming conventions as for all other endpoints in this schema e.g. getUserFlexitime